markdownx/tests/tests.py

```diff
@@ -1,8 +1,35 @@
 from django.test import TestCase
+try:
+    # Django >= 2.0
+    from django.urls import reverse
+except ImportError:
+    from django.core.urlresolvers import reverse
+import os
+import re
 
 
 class SimpleTest(TestCase):
 
     def test_me(self):
         response = self.client.get('/testview/')
         self.assertEqual(response.status_code, 200)
+
+    def test_upload(self):
+        url = reverse('markdownx_upload')
+
+        with open('markdownx/tests/static/django-markdownx-preview.png', 'rb') as fp:
+            response = self.client.post(url, {'image': fp}, HTTP_X_REQUESTED_WITH='XMLHttpRequest')
+
+            try:
```

Comment:
I think the rest of the test can be deindented. It doesn't need to be in the "with" context manager.